pristine-zen-garden.html

```diff
@@ -0,0 +1,157 @@
+        <p>We ask that you submit original artwork. Please respect copyright laws. Please keep objectionable material to a minimum, and try to incorporate unique and interesting visual themes to your work. We&#8217;re well past the point of needing another garden-related design.</p>
+        <p>This is a learning exercise as well as a demonstration. You retain full copyright on your graphics (with limited exceptions, see <a href="http://www.mezzoblue.com/zengarden/submit/guidelines/">submission guidelines</a>), but we ask you release your <abbr title="Cascading Style Sheets">CSS</abbr> under a Creative Commons license identical to the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Zen Garden's license information.">one on this site</a> so that others may learn from your work.</p>
+        <p role="contentinfo">By <a href="http://www.mezzoblue.com/">Dave Shea</a>. Bandwidth graciously donated by <a href="http://www.mediatemple.net/">mediatemple</a>. Now available: <a href="http://www.amazon.com/exec/obidos/ASIN/0321303474/mezzoblue-20/">Zen Garden, the book</a>.</p>
+      </div>
+
+      <footer>
+        <a href="http://validator.w3.org/check/referer" title="Check the validity of this site&#8217;s HTML" class="zen-validate-html">HTML</a>
+        <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Check the validity of this site&#8217;s CSS" class="zen-validate-css">CSS</a>
+        <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Creative Commons license of this site: Attribution-NonCommercial-ShareAlike." class="zen-license">CC</a>
+        <a href="http://mezzoblue.com/zengarden/faq/#aaa" title="Read about the accessibility of this site" class="zen-accessibility">A11y</a>
+        <a href="https://github.com/mezzoblue/csszengarden.com" title="Fork this site on Github" class="zen-github">GH</a>
+      </footer>
+
+    </div>
+
+
+    <aside class="sidebar" role="complementary">
+      <div class="wrapper">
+
+        <div class="design-selection" id="design-selection">
+          <h3 class="select">Select a Design:</h3>
+          <nav role="navigation">
+            <ul>
+              <li>
+                <a href="/205/" class="design-name">spring360</a> by <a href="http://www.medialab360.com" class="designer-name">Rene Hornig</a>
+              </li>         <li>
+              <a href="/204/" class="design-name">Withering Beauty</a> by     <a href="http://www.wduffy.co.uk/" class="designer-name">William Duffy</a>
+            </li>         <li>
+            <a href="/203/" class="design-name">Tiny Blue</a> by            <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a>
+          </li>         <li>
+          <a href="/202/" class="design-name">Retro Theater</a> by            <a href="#" class="designer-name">Eric Rogé</a>
```

Comment:
`<li>Bullet points</li>`
`<li>Should be aligned</li>`
`<li>but indented together</li>`
`<li>because they're nested</li>`
'<li>under the <ul></ul> tags</li>`